### PR TITLE
Adding new variables required for identity management

### PR DESCRIPTION
**Pull request by makirill**

### CODE REVIEW

File: inventory-generation/identity-management/main.yml [hunk 1]
```diff
@@ -9,6 +9,10 @@
       when:
         - directory is undefined or (directory | trim) == ""
 
+    - name: Read Engagement Data
```
Comment (oybed) [inline inventory-generation/identity-management/main.yml:12]: This move seems unnecessary - unless there's a reason I'm missing, please undo this change

**Review by oybed — CHANGES_REQUESTED** [review]

File: inventory-generation/identity-management/main.yml (hunk 1 above)
Comment (makirill) [reply inventory-generation/identity-management/main.yml:12]: The `governor_spec` is defined using `engagement_region` (see [lodestar-deployment](https://github.com/rht-labs/lodestar-deployment/pull/368/files)), so we needed to have `engagement_region` variable defined before check that`governor_spec` defined.

Not a best solution, and probably should be done some other way.

Comment (oybed) [reply inventory-generation/identity-management/main.yml:12]: OK, great - thx for the explanation

**Review by oybed — APPROVED** [review]
